src/WingetCreateCore/Common/PackageParser.cs

```diff
@@ -432,10 +445,29 @@ private static void UpdateInstallerMetadata(Installer existingInstaller, Install
             existingInstaller.PackageFamilyName = newInstaller.PackageFamilyName ?? existingInstaller.PackageFamilyName;
             existingInstaller.NestedInstallerFiles = newInstaller.NestedInstallerFiles ?? existingInstaller.NestedInstallerFiles;
             existingInstaller.Platform = newInstaller.Platform ?? existingInstaller.Platform;
+
+            if (existingInstaller.AppsAndFeaturesEntries != null && newInstaller.AppsAndFeaturesEntries != null)
+            {
+                // New installer will always have a single entry in AppsAndFeaturesEntries

```

Comment:
nit: Can you add to this comment explaining that the AppsAndFeaturesEntries for the new installer will not be null if there is a display-version argument present.